doc/releases/release-notes-3.6.rst

```diff
@@ -244,36 +284,51 @@ Drivers and Sensors
 
   * Renesas R-Car clock control driver now supports Gen4 SoCs
   * Renamed ``CONFIG_CLOCK_CONTROL_RA`` to :kconfig:option:`CONFIG_CLOCK_CONTROL_RENESAS_RA`
+  * On STM32 devices, :dtcompatible:`st,stm32-hse-clock` no allows setting a ``css-enabled``
+    property which enables HSE clock security system (CSS).
 
 * Counter
 
   * The nRFx counter driver now works with simulated nrf*_bsim targets.
 
   * counter_native_posix driver: Added support for top value configuration, and a bugfix.
 
+* Crypto
+
+  * STM32WB devices now support crypto API through AES block
+
 * DAC
 
 * Disk
 
 * Display
 
+  * Introduce frame buffer config to STM32 LTDC driver.
+
 * DMA
 
+  * STM32WBA Devices now support GPDMA
+
+* EEPROM
```

Comment:
This needs not be added.

Reply:
Yeah, this is coming from a rebase, not sure how it happened.